test/test_getting_tables.py

```diff
@@ -666,3 +666,76 @@ def test_insert_into_on_duplicate_key_ipdate():
         " VALUES ('user1', 'john doe', 20)"
         " ON DUPLICATE KEY UPDATE name='john doe', age=20"
     ).tables == ["user"]
+
+
```

Comment:
@Byunk - can you please reformat that file using `black`?

Reply:
@macbre sure. please check it!